tests/Tests/200__monitor_and_manage/200__metrics/203__alerts.robot

```diff
@@ -20,10 +19,11 @@ ${notebook_clean}=  /ods-ci-notebooks-main/notebooks/200__monitor_and_manage/203
 
 *** Test Cases ***
 Verify alert RHODS-PVC-Usage is fired when user notebook pvc usage is above 90 Percent
-  [Tags]  Sanity  ODS-516 
+  [Tags]  Sanity  ODS-516
   Set Up Alert Test  ${notebook_90}
   Sleep  320
   Prometheus.Alert Should Be Firing  ${RHODS_PROMETHEUS_URL}  ${RHODS_PROMETHEUS_TOKEN}  ${rule_group}  ${alert_90}
+  [Teardown]  Clean Up Files And End Web Test
```

Comment:
Have you confirmed that the clean up process is not causing the 403 error popup?

Reply:
I believe the errors we saw were because the PVC was filled 100% and the jupyterhub database file was blocked. Now the tests leave a little bit of free space so this doesn't happen.

I haven't seen the error 403 in the pr jobs 239 and 240